src/test/java/teammates/test/BaseTestCaseWithObjectifyAccess.java

```diff
@@ -59,6 +59,11 @@ public void resetLocalDatastoreHelper() throws IOException {
         localDatastoreHelper.reset();
     }
 
+    @AfterClass
+    public void resetSearchService() {
+        SearchManagerFactory.resetSearchManager();
```

Comment:
You would have probably noticed by now, but this method needs to be suppressed/overridden in E2E tests.